pkg/distributor/otel.go

```diff
@@ -710,8 +710,8 @@ func TimeseriesToOTLPRequest(timeseries []prompb.TimeSeries, metadata []mimirpb.
 				if i == 0 {
 					for _, tsEx := range ts.Exemplars {
 						ex := datapoint.Exemplars().AppendEmpty()
-						ex.SetDoubleValue(histogram.Sum / 10.0) // Doesn't really matter, just a placeholder
-						ex.SetTimestamp(pcommon.Timestamp(histogram.Timestamp * time.Millisecond.Nanoseconds()))
+						ex.SetDoubleValue(tsEx.Value)
+						ex.SetTimestamp(pcommon.Timestamp(tsEx.Timestamp * time.Millisecond.Nanoseconds()))
```

Comment:
Note to reviewers: this looks a bug. We have to preserve the histogram's exemplar value and timestamp , like we do for float samples. However, this function is just used in tests, so not affecting customers.